File: A1/A1-010/A1-010.py
# --------------------------------------------------
# File Name : A1-010.py
# Problem   : Ticket
# Author    : Worralop Srichainont
# Date      : 2026-02-08
# --------------------------------------------------

# Input
age = int(input())
ticket_type = input().strip()

# Output
if age < 18 or ticket_type.lower() == "s":
    print(20)
else:
    print(50)

File: A1/A1-010/A1-010.cpp
#include<iostream>
using namespace std;

int main()
{
    // This makes std::cin and std::cout faster
    ios_base::sync_with_stdio(false); cin.tie(0);

    // Input
    int age;
    char type;
    cin >> age >> type;
    // Output
    if(age < 18 || type == 's' || type == 'S')
    {
        cout << 20;
    }
    else { cout << 50; }
    return 0;
}